server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -57,7 +57,7 @@
 import io.crate.testing.UseRandomizedOptimizerRules;
 import io.crate.testing.UseRandomizedSchema;
 
-@IntegTestCase.ClusterScope(numDataNodes = 2)
+@IntegTestCase.ClusterScope(minNumDataNodes = 2)
```

Comment:
Was it a temporal change for debugging or we need >2 nodes to provoke the error?

Reply:
>2 makes it more likely and didn't really see a reason to keep it limited to 2